### PR TITLE
Customize content, site landing page, and CMS collections

### PR DESCRIPTION
**Pull request by alexearnshaw**
## Describe the changes

This PR replaces the placeholder content with the AMPSS content and updates the Netlify CMS config with collections to match the AMPSS content. I've also made some changes to the AMPSS Documentation home page to explain the open beta (as planned in ticket RMF-5)

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap as mentioned I've changed the documentation home page for this microsite to explain the open beta. This is planned in RMF-5, but as it will need to be done in each microsite now I decided to do it in this PR for Shared Services. Here is the updated page https://deploy-preview-7--amplifysharedservices-open-docs.netlify.app/docs/

File: content/en/docs/_index.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
+Axway AMPLIFY Shared Services documentation is available for collaboration using an open source model as part of the Axway Open Documentation Open Beta program.
 
-This section is where the user documentation for your project lives - all the information your users need to understand and successfully use your project.
+## About the Open Beta program
 
-For large documentation sets we recommend adding content under the headings in this section, though if some or all of them don’t apply to your project feel free to remove them or add your own.
\ No newline at end of file
+The Open Beta enables you to suggest edits and create issues for any page of the documentation using a GitHub account. To learn more, see the [contribution guidelines](/docs/contribution_guidelines/).
```
Comment (andreamussap) [inline content/en/docs/_index.md:15]: This link is broken.  The contrib guidelines is going to be one single file, stored on axway-open-docs, or one version for each microsite? I think one unique file would be better. So in this case might have to specify the full path - https://axway-open-docs.netlify.app/docs/contribution_guidelines/ ??

File: content/en/docs/_index.md [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+
+The role of each documentation website in the Open Beta is as follows:
+
+* Axway Open Documentation on Netlify (<https://axway-open-docs.netlify.app/>) - Specific product documentation sets are available on this website for collaboration using GitHub and Netlify CMS.
```
Comment (andreamussap) [inline content/en/docs/_index.md:19]: on this website for **contribution**  ?? I think 'collaboration' is ok on the fist paragraph, gives a sense of  whole, but here where it's more specific, 'contribution' reads better IMHO

**Comment by andreamussap** [comment]
> @andreamussap ...Here is the updated page https://deploy-preview-7--amplifysharedservices-open-docs.netlify.app/docs/

@alexearnshaw I've left two minor comments. I think the overview is very good, and we can use the same "template" for the other microsites ??

File: content/en/docs/_index.md (hunk 1 above)
Comment (alexearnshaw) [reply content/en/docs/_index.md:15]: This links is also broken on the right nav bar and I'd like to check this with @winghonlai as I think these links should work without having to put the full path, as the links in the header to blog and community work even though that content is not in this repo but in the common repo also.
I agree all sites should be using a common file.

File: content/en/docs/_index.md (hunk 2 above)
Comment (alexearnshaw) [reply content/en/docs/_index.md:19]: agree thanks will change

**Comment by alexearnshaw** [comment]
> > @andreamussap ...Here is the updated page https://deploy-preview-7--amplifysharedservices-open-docs.netlify.app/docs/
> 
> @alexearnshaw I've left two minor comments. I think the overview is very good, and we can use the same "template" for the other microsites ??

Thanks, yes I agree we should use the same template for all microsites

**Pull request merged**